test/index.js

```diff
@@ -155,7 +155,7 @@ describe('onBuild()', () => {
     expect(await pathExists(PUBLISH_DIR)).toBeTruthy()
   })
 
-  test('calls copySync with correct args', async () => {
+  test('copy files to the publish directory', async () => {
```

Comment:
Some unused fixture files for this test were accidentally committed to `git` in a previous PR.